### PR TITLE
Update readme after RFC review meeting 

### PR DESCRIPTION
**Pull request by juliandescottes**
- removed some files that are no longer relevant with the simplified repo
- updated the repo to mention the main changes we discussed during the meeting:
  - send an email as a heads up before the review meeting so that peers can comment
  - use labels to keep track of what was accepted or rejected

### CODE REVIEW

**Comment by juliandescottes** [comment]
cc @bgrins @gregtatum @jryans

File: README.md [hunk 1]
```diff
@@ -21,16 +25,18 @@ The list of RFCs currently under discussion can be found at: [Active RFC List][]
 
 Each week the [DevTools core team][] will attempt to review some set of open RFC issues. Reviewed RFCs will also be mentioned in the weekly DevTools team meeting (see archives for [devtools-weekly-meetings][]).
```
Comment (gregtatum) [inline README.md]: Are these supposed to be actual links?

Comment (juliandescottes) [reply README.md]: Yes, the link locations are defined at the bottom. I don't usually do that in md files, but that's how links were used in this one so I kept following this pattern.

Comment (gregtatum) [reply README.md]: https://github.com/juliandescottes/rfcs/tree/readme `[DevTools core team][]` isn't clickable here.

**Comment by juliandescottes** [comment]
> Should we delete the package.json and .gitignore as well? It seems like we don't need them anymore. 

Yes, thanks! done.

File: README.md (hunk 1 above)
Comment (juliandescottes) [reply README.md]: ah! there was a typo in this one, should have been `DevTools RFC team`, good catch.

**Pull request merged**